### DISCUSSION TITLE
File Being Written No Longer Visible

### DISCUSSION THREAD

**Discussion by THRobinson75** [RandomEngy/VidCoder#1282] [General]
While encoding to the destination folder, it use to show the file with part or something at the end to show it wasn't finished. Now, it's just not there until it is finished. Not sure where the temp file is anymore or how to get it back. I found it handy to see because sometimes you could see the file size vs percentage complete and know it's going to be too big so, may as well cancel and adjust settings.

Not sure if coincidental or not, but same time that vanished, I noticed my screen saver no longer kicks in when encoding. Not sure if a vidcoder issue or not.

Current 11.4 Beta

**Comment by THRobinson75** [comment]
Nevermind, I think I figured it out. 

Have a batch that I am doing at avg bitrate... normally I do constant quality but old grainy DVD stuff I find works better sometimes with avg bitrate and 2-pass. I guess it's the reason,

**Comment by RandomEngy** [comment]
VidCoder declares that it needs the system to stay awake while encoding, but it doesn't ask for the screen to stay on. I don't think this would be caused by VidCoder.

**Comment by THRobinson75** [comment]
Ya, I'm not sure what was the culprit there... before I realized why the temp files were missing (2pass CBR) I reverted back to stable instead of beta/nightly.... coincidence? maybe? but screensaver works again.
